<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentSearching</class>
 <widget class="QDialog" name="studentSearching">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student searching</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/searchWindow.png</normaloff>:/icons/searchWindow.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="IDArea">
         <property name="placeholderText">
          <string>Student ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkIDBtn">
         <property name="text">
          <string>Check ID</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="firstYearBtn">
         <property name="text">
          <string>First year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="secondYearBtn">
         <property name="text">
          <string>Second year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="thirdYearBtn">
         <property name="text">
          <string>Third Year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fourthYearBtn">
         <property name="text">
          <string>Fourth year</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="firstSemesterBtn">
         <property name="text">
          <string>First semester</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="secondSemesterBtn">
         <property name="text">
          <string>Second semester</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="yearAndSemesterLabel">
           <property name="text">
            <string>Year and Semester</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="studentNameLabel">
           <property name="text">
            <string>Student name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="studentGPALabel">
           <property name="text">
            <string>Student GPA</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="subjectsArea">
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
